include/wil/process_helpers.h

```diff
@@ -0,0 +1,208 @@
+//*********************************************************
+//
+//    Copyright (c) Microsoft. All rights reserved.
+//    This code is licensed under the MIT License.
+//    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF
+//    ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+//    TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+//    PARTICULAR PURPOSE AND NONINFRINGEMENT.
+//
+//*********************************************************
+//! @file
+//! Helpers for interfacing with process snapshots (e.g. enumerating processes)
+#ifndef __WIL_PROCESS_HELPERS_INCLUDED
+#define __WIL_PROCESS_HELPERS_INCLUDED
+
+#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
+
+#if (__WI_LIBCPP_STD_VER >= 17) && WI_HAS_INCLUDE(<string_view>, 1) // Assume present if C++17
+#if defined(WIL_ENABLE_EXCEPTIONS)
+
+#if __cpp_lib_string_view >= 201606L

```

Comment:
```suggestion
#include "common.h"

#if WIL_USE_STL && (__WI_LIBCPP_STD_VER >= 17) && WI_HAS_INCLUDE(<string_view>, 1) // Assume present if C++17
#include <string_view>
#endif

#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP) && defined(WIL_ENABLE_EXCEPTIONS) && (__cpp_lib_string_view >= 201606L)
```

Use `WIL_USE_STL` to determine if it's okay to use the STL.

Need to include `common.h` for `WIL_ENABLE_EXCEPTIONS` and `WIL_USE_STL` definitions.

Need to include `string_view` to get the definition of `__cpp_lib_string_view`